lib/slop/result.rb

```diff
@@ -20,6 +20,11 @@ def [](flag)
     end
     alias get []
 
+    # Returns an options value, raises KeyError if the option does not exist.
```

Comment:
`options` => `option's`

Reply:
Thanks, fixed here and in `Slop::Result#[]` description!